src/main/scala/djinni/generator.scala

```diff
@@ -704,36 +746,50 @@ abstract class Generator(spec: Spec) {
 
   def normalEnumOptions(e: Enum) = e.options.filter(_.specialFlag.isEmpty)
 
-  def writeEnumOptionNone(w: IndentWriter, e: Enum, ident: IdentConverter) {
-    for (
-      o <- e.options.find(_.specialFlag.contains(Enum.SpecialFlag.NoFlags))
-    ) {
+  def writeEnumOptionNone(
+      w: IndentWriter,
+      e: Enum,
+      ident: IdentConverter,
+      delim: String = "="
+  ) {
+    for (o <- e.options.find(_.specialFlag == Some(Enum.SpecialFlag.NoFlags))) {
       writeDoc(w, o.doc)
-      w.wl(ident(o.ident.name) + " = 0,")
+      w.wl(ident(o.ident.name) + s" $delim 0,")
     }
   }
 
-  def writeEnumOptions(w: IndentWriter, e: Enum, ident: IdentConverter) {
+  def writeEnumOptions(
+      w: IndentWriter,
+      e: Enum,
+      ident: IdentConverter,
+      delim: String = "="
+  ) {
     var shift = 0
     for (o <- normalEnumOptions(e)) {
       writeDoc(w, o.doc)
-      w.wl(ident(o.ident.name) + (if (e.flags) s" = 1 << $shift" else "") + ",")
+
+      if (e.flags) {
+        w.wl(ident(o.ident.name) + s" $delim 1 << $shift")
+      } else {
+        w.wl(ident(o.ident.name) + s" $delim $shift" + ",")
+      }
+
```

Comment:
I cannot locate the exact spot but some of these changes to the enum output generation must be the reasons why the tests fail. Pls make sure to only include changes that are directly related to WASM-support!

Reply:
it is outdated and was solved